src/lib.rs

```diff
@@ -1,19 +1,39 @@
 //! # Oxc Resolver
 //!
-//! Node.js Module Resolution.
+//! Node.js [CommonJS][cjs] and [ECMAScript][esm] Module Resolution.
 //!
-//! All configuration options are aligned with [enhanced-resolve].
+//! A module resolution is the process of finding the file referenced by a module specifier in
+//! `import "specifier"` or `require("specifier")`.
+//!
+//! All [configuration options](ResolveOptions) are aligned with webpack's [enhanced-resolve].
+//!
+//! ## Terminology
+//!
+//! ### Specifier
+//!
+//! For [CommonJS modules][cjs],
+//! the specifier is the string passed to the `require` function. e.g. `"id"` in `require("id")`.
+//!
+//! For [ECMAScript modules][esm],
+//! the specifier of an `import` statement is the string after the `from` keyword,
+//! e.g. `'node:path'` in `import { sep } from 'node:path'`.
```

Comment:
```suggestion
//! e.g. `'node:path'` in `import { sep } from 'node:path'`, or `'bootstrap.css'` in `import 'bootstrap.css'`
```

Added suggestion as per my previous commet